modules/host/negotiate.go

```diff
@@ -75,7 +75,7 @@ func (h *Host) considerTerms(terms modules.ContractTerms) error {
 	case len(terms.MissedProofOutputs) != 1:
 		return errors.New("refund len does not match host settings")
 
-	case terms.MissedProofOutputs[0].UnlockHash != types.ZeroUnlockHash:
+	case terms.MissedProofOutputs[0].UnlockHash != types.UnlockHash{}:
```

Comment:
why the change? Explicitly naming ZeroUnlockHash makes the code more readable (and prevents the rare, but annoying syntax of `if x != (types.UnlockHash{})` (note parens)